cargo/flatpak-cargo-generator.py

```diff
@@ -74,6 +74,28 @@ async def get_remote_sha256(url):
     return sha256.hexdigest()
 
 
+def find_toml(git_repo_dir):
+    if os.path.isfile(os.path.join(git_repo_dir, 'Cargo.toml')):
+        return load_toml(os.path.join(git_repo_dir, 'Cargo.toml')), '.'
+    # If Cargo.toml isn't found in repository root, scan subdirectories
+    toml_list = []
+    for entry in os.scandir(git_repo_dir):
+        if entry.is_dir():
+            toml_path = os.path.join(git_repo_dir, entry.name, 'Cargo.toml')
```

Comment:
I guess this will work, too?
```suggestion
            toml_path = os.path.join(entry.path, 'Cargo.toml')
```